pyvo/registry/tests/test_rtcons.py

```diff
@@ -284,6 +284,35 @@ def test_basic(self):
                     "ucd LIKE 'phot.mag;em.opt.%' OR ucd LIKE 'phot.mag;em.ir.%'"))
 
 
+@pytest.mark.usefixtures('uat_vocabulary')
+class TestUATConstraint:
+    def test_basic(self):
+        cons = rtcons.UAT("solar-flares")
+        assert (cons.get_search_condition(FAKE_GAVO)
```

Comment:
Can we have some tests that query the remote service rather than the mock version? (OTOH, if you say the doctests should be enough for that, I would be signing off on it)